cobalt/build/gn.py

```diff
@@ -30,7 +30,8 @@
     },
     'devel': {
         'symbol_level': 1,
-        'is_debug': 'false'
+        'is_debug': 'false',
+        'dcheck_always_on': 'true',  # On by default if `is_debug`.
```

Comment:
This already defaults to true for devel builds

Reply:
> This already defaults to true for devel builds

Oh! I must have broken something in my GNs, it wasn't
when I uploaded it. Never mind! Thanks!